src/effects/Effect.cpp

```diff
@@ -155,18 +155,17 @@ int Effect::ShowClientInterface(wxWindow &parent, wxDialog &dialog,
    EffectUIValidator *, bool forceModal)
 {
    // Remember the dialog with a weak pointer, but don't control its lifetime
```

Comment:
Comment appears to be out of date

Reply:
I think I removed it at some late commit in this long sequence?